src/javascript/app/pages/user/new_account/virtual_acc_opening.js

```diff
@@ -96,6 +99,12 @@ const VirtualAccOpening = (() => {
 
         if (Cookies.get('affiliate_tracking')) req.push({ request_field: 'affiliate_token', value: Cookies.getJSON('affiliate_tracking').t });
 
+        const mobile_devices = ['iPhone', 'iPad', 'Android'];
+        const date_first_contact = LocalStore.get('date_first_contact') != null ?
+            LocalStore.get('date_first_contact') :
+            toISOFormat(moment());
```

Comment:
would `LocalStore.get('date_first_contact') || toISOFormat(moment())` be the same?